apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -2224,19 +2224,19 @@
       },
       "SERVICE_IDENTITIES": {
         "TITLE": "Service accounts",
-        "CREATE": "Add",
-        "DELETE": "Remove",
-        "NO_IDENTITIES": "No service accounts assigned."
+        "CREATE": "Connect",
+        "DELETE": "Disconnect",
+        "NO_IDENTITIES": "No service accounts connected."
       },
       "GUI_CONFIG": {
         "PREF_TABLE_PAGE_SIZE": "Preferred table page size",
         "TITLE": "Gui Configuration"
       },
       "ASSOCIATED_USERS": {
-        "TITLE": "Associated users",
-        "ADD": "Add",
-        "REMOVE": "Remove",
-        "NO_ASSOCIATED_USERS": "No associated users assigned."
+        "TITLE": "Account managers",
+        "ADD": "Connect account",
+        "REMOVE": "Disconnect account",
```

Comment:
I dont think we need the word "account" in those buttons
in the remove button I think the remove its just fine
maybe the add here was also just fine